Cubical/Data/Int/MoreInts/DiffInt/Properties.agda

```diff
@@ -52,3 +56,172 @@ private
+    fst z +ℕ fst y +ℕ (snd x +ℕ snd z)   ≡⟨ cong (λ x' → x' +ℕ (snd x +ℕ snd z)) (ℕ.+-comm (fst z) (fst y)) ⟩
+    fst y +ℕ fst z +ℕ (snd x +ℕ snd z) ∎
+
+ℤ-isSet : isSet ℤ
+ℤ-isSet m n p q = squash/ m n p q
+
+_+_ : ℤ → ℤ → ℤ
+_+_ = SetQuotients.rec2 ℤ-isSet (λ x y → [ x +ℕ' y ]) feql feqr
+  where
+    feql : (a b c : ℕ × ℕ) (r : rel a b) → [ a +ℕ' c ] ≡ [ b +ℕ' c ]
+    feql a b c r = eq/ (a +ℕ' c) (b +ℕ' c) (+-left-congruence a b c r)
+    feqr : (a b c : ℕ × ℕ) (r : rel b c) → [ a +ℕ' b ] ≡ [ a +ℕ' c ]
+    feqr a b c r = eq/ (a +ℕ' b) (a +ℕ' c) (+-right-congruence a b c r)
+
+private
+  test : [ 3 , 9 ] + [ 10 , 14 ] ≡ [ 0 , 10 ]
+  test = ℤ-cancelˡ 13
+
+  zero-identityˡ-lem : (a : ℕ)(b : ℤ) → (Σ (ℕ × ℕ) ( λ (c , d) → [ (c , d) ]  ≡ b )) → [ a , a ] + b ≡ b
+  zero-identityˡ-lem a b ( (c , d) , p ) =
+    [ a , a ] + b           ≡⟨ cong (λ x → [ a , a ] + x) (sym p) ⟩
+    [ a , a ] + [ c , d ]   ≡⟨ ℤ-cancelˡ a ⟩
+    [ c , d ]               ≡⟨ p ⟩
+    b ∎
+
+  zero-identityʳ-lem : (a : ℕ)(b : ℤ) → (Σ (ℕ × ℕ) ( λ (c , d) → [ (c , d) ]  ≡ b )) → b + [ a , a ] ≡ b
+  zero-identityʳ-lem a b ( (c , d) , p ) =
+    b + [ a , a ]           ≡⟨ cong (λ x → x + [ a , a ]) (sym(p)) ⟩
+    [ c , d ] + [ a , a ]   ≡⟨ ℤ-cancelʳ a ⟩
+    [ c , d ]               ≡⟨ p ⟩
+    b ∎
+
+
+zero-identityˡ : (a : ℕ) (b : ℤ) → [ a , a ] + b ≡ b
+zero-identityˡ a b = PropositionalTruncation.rec (lem2 a b) (zero-identityˡ-lem a b) ([]surjective b)
+                                               where
+                                                 lem2 : (a : ℕ) (b : ℤ) → isProp ([ a , a ] + b ≡ b)
+                                                 lem2 a b = ℤ-isSet ([ a , a ] + b) b
+
+zero-identityʳ : (a : ℕ)(b : ℤ) → b + [ a , a ] ≡ b
+zero-identityʳ a b = PropositionalTruncation.rec (lem2 a b) (zero-identityʳ-lem a b) ([]surjective b)
+                                               where
+                                                 lem2 : (a : ℕ) (b : ℤ) → isProp (b + [ a , a ] ≡ b)
+                                                 lem2 a b = ℤ-isSet (b + [ a , a ]) b
+
+private
+  -ℤ'_  : ℤ → ℤ
+  -ℤ' [ a ] = [ snd a , fst a ]
+  -ℤ' eq/ a b r i = eq/ (snd a , fst a) (snd b , fst b) (tmp a b r) i
+                                                        where
+                                                          tmp : ∀ a b → fst a +ℕ snd b ≡ fst b +ℕ snd a → snd a +ℕ fst b ≡ snd b +ℕ fst a
+                                                          tmp a b r =
+                                                            snd a +ℕ fst b ≡⟨ ℕ.+-comm (snd a) (fst b) ⟩
+                                                            fst b +ℕ snd a ≡⟨ sym r ⟩
+                                                            fst a +ℕ snd b ≡⟨ ℕ.+-comm (fst a) (snd b) ⟩
+                                                            snd b +ℕ fst a ∎
+  -ℤ' squash/ a a₁ p q i i₁ = squash/ (-ℤ' a) (-ℤ' a₁) (cong (λ x → -ℤ' x) p) (cong (λ x → -ℤ' x) q) i i₁
```

Comment:
You should avoid the long line by moving the 'where'-block more to the left.

Reply:
Thanks for the comment,
I have fixed it now.